.bumpversion.cfg

```
@@ -1,12 +1,14 @@
 [bumpversion]
 current_version = 1.5.0a1
-parse = (?P<major>\d+)
-	\.(?P<minor>\d+)
-	\.(?P<patch>\d+)
-	((?P<prekind>a|b|rc)
-	(?P<pre>\d+)  # pre-release version num
-	)(\.(?P<nightly>[a-z..0-9]+)
+parse = (?P<major>[\d]+)
+	\.(?P<minor>[\d]+)
+	\.(?P<patch>[\d]+)
+	(((?P<prekind>a|b|rc)
+	?(?P<pre>[\d]+?))
+	\.?(?P<nightly>[a-z..0-9]+\+[a-z]+)?
```

```suggestion
	\.?(?P<nightly>[a-z0-9]+\+[a-z]+)?
```
The two dots in this character class definition are literals, so this would match `1.5.0a1.dev123.457+nightly` as well. If you'd like to match on dots, remove the duplicate.